_sass/partials/_statistic.scss

```diff
@@ -6,7 +6,7 @@
 	}
     .stat {
     	font-size: 100px;
-	    color: $link-color;
+	    color: #f4f7fb;
```

Comment:
Makes sense to have a variable name for this color be quickly changed/reused in other instances, the same way we had previously with `$link-color` (probably another name that would make sense here)?